src/ports/Jwt.ts

```diff
@@ -0,0 +1,106 @@
+import cookie from 'cookie'
+import { CrossFetchRequest, Fetcher } from 'dcl-catalyst-commons'
+import log4js from 'log4js'
+import ms from 'ms'
+import { generateNonceForChallenge } from '../utils/ProofOfWork'
+
+const LOGGER = log4js.getLogger('JWTPort')
+
+export async function obtainJWT(fetcher: Fetcher, catalystUrl: string): Promise<string | undefined> {
+  try {
+    const response = await fetcher.fetchJson(catalystUrl + '/pow-auth/challenge')
+    const body = JSON.parse(JSON.stringify(response))
```

Comment:
Is this a cheap `clone(response)`?

Reply:
Oops, that line shouldn't be necessary. Nice catch!